src/ClusterBootstrap/services/job-insighter/job-insighter-cronjob.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: job-insighter-cronjob
+  labels:
+    app: job-insighter
+spec:
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    metadata:
+      name: job-insighter-job
+      labels:
+        app: job-insighter
+    spec:
+      activeDeadlineSeconds: 900
+      template:
+        metadata:
+          name: job-insighter-pod
+          labels:
+            app: job-insighter
+        spec:
+          containers:
+          - name: job-insighter
+            image: '{{cnf["worker-dockerregistry"]}}/{{cnf["dockerprefix"]}}/job-insighter:{{cnf["dockertag"]}}'
+            imagePullPolicy: Always
+            # Job insighter requires that restful server and prometheus run on the infra node.
+            args:
+              - 'python'
+              - '/job-insighter/insight.py'
+              - '--restful_url'
+              - 'http://localhost:{{cnf["restfulapiport"]}}'
+              - '--prometheus_url'
+              - 'http://localhost:{{cnf["prometheus"]["port"]}}'
+          {% if cnf["private_docker_registry_username"] %}
+          imagePullSecrets:
+          - name: svccred
+          {% endif %}
+          nodeSelector:
+            job-insighter: active
+          hostNetwork: true
+          restartPolicy: Never
+          tolerations:
+          - key: node-role.kubernetes.io/master
+            effect: NoSchedule
+  schedule: '*/15 * * * *'

```

Comment:
What's the behaviour of previous job still running when the next job is about to start? Will two jobs run concurrently? I see `concurrencyPolicy: Forbid`, will next job start running right away after the previous job finished or wait for more time?

Reply:
The one second one will not be able to start and be counted as a miss.
https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/